supplementary_style_guide/introduction/whats-new.adoc

```diff
@@ -32,6 +32,23 @@ To view the history of changes from 2020 through 2022, see the link:https://gith
 [[year-2024]]
 == 2024
 
+[[july-2024]]
+=== July 2024
+
+.Glossary entries
+* *xref:composite-content-view[composite content view]*: Updated glossary entry to lowercase spelling
+* *xref:content-view[content view]*: Updated glossary entry to lowercase spelling
+* *xref:nmstate[Nmstate]*: Added glossary entry
+* *xref:on-premise[on-premise]*: Updated glossary entry to include Red Hat OpenShift Container Platform definition
+* *Product*: Removed glossary entry
+* *Red{nbsp}Hat Network Proxy Server*: Removed glossary entry
+* *ed{nbsp}Hat Network Satellite Server*: Removed glossary entry
+* *xref:subscription-manifest[subscription manifest]*: Updated glossary entry to lowercase spelling
+
+.Style guidance
+* *xref:external-links[External links]*: Added guidance to avoid using URL shorteners.
```

Comment:
I think we've been using no punctuation on these (though I do see some with them - probably could remove throughout any w/ them)

```suggestion
* *xref:external-links[External links]*: Added guidance to avoid using URL shorteners
```